src/applications/caregivers/components/FormFields/FacilityList.jsx

```diff
@@ -42,7 +42,7 @@ const FacilityList = props => {
       key={f.id}
       name="facility"
       label={f.attributes.name}
-      value={f.id.split('_').pop()}
+      value={f.id}
```

Comment:
We no longer need to strip the facility id here. We are now doing this before submission in the submit-transformer.